pihole

```diff
@@ -481,41 +482,41 @@ Options:
   echo -e "${OVER}  ${TICK} ${str}"
 }
 
-piholeStatus() {
-  if [[ "$(netstat -plnt | grep -c ':53 ')" -gt "0" ]]; then
-    if [[ "${1}" != "web" ]]; then
-      echo -e "  ${TICK} DNS service is running"
-    fi
+statusFunc() {
+  local addnConfigs
+
+  # Determine if service is running on port 53
+  if nc -z 127.0.0.1 53; then
```

Comment:
`if [[ nc -z 127.0.0.1 53 ]]; then`

But I'm not sure if this is just goal post stuff, `nc` isn't stock on a lot of systems. Some have BSD `netcat` and some don't have anything installed at all. Will have to check a few systems to make sure this is available.

Reply:
I'll be happy to sort the style issue out once there's more word on this. It'll also mean [this code](https://github.com/pi-hole/pi-hole/blob/feb412b02af9b0b0465e096f4141ac0d23e7349d/pihole#L564) will need to be changed.